### ISSUE TITLE
import imp is deprecated

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#8702]
|[<img alt="rodrigc's avatar" src="https://avatars.githubusercontent.com/u/1895943?s=50" width="50" height="50">](https://github.com/rodrigc)| @rodrigc reported|
|-|-|
|Trac ID|trac#8702|
|Type|enhancement|
|Created|2016-07-28 19:10:01Z|
|Branch|https://github.com/twisted/twisted/tree/8702-rodrigc-imp|

I ran the following with Python 3.5

```
python -Wall -bb bin/trial twisted.python.test.test_runtime
```

and saw this:

```
twisted/python/runtime.py:10: DeprecationWarning: the imp module is deprecated in favour of importlib; see the module's documentat
  import imp
```

<details><summary>Searchable metadata</summary>

```
trac-id__8702 8702
type__enhancement enhancement
reporter__rodrigc rodrigc
priority__normal normal
milestone__Python_3_x Python-3.x
branch__8702_rodrigc_imp 8702-rodrigc-imp
branch_author__ 
status__closed closed
resolution__fixed fixed
component__core core
keywords__None None
time__1469733001482514 1469733001482514
changetime__1474198049187875 1474198049187875
version__None None
owner__GitHub__noreply_____ GitHub <noreply@...>

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="rodrigc's avatar" src="https://avatars.githubusercontent.com/u/1895943?s=50" width="50" height="50">](https://github.com/rodrigc)<a name="note_1"></a>|@rodrigc commented|
|-|-|

See:
https://github.com/twisted/twisted/pull/439

**Comment by twisted-trac** [comment]
|<img alt="GitHub's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50"><a name="note_2"></a>|GitHub set owner to GitHub<br>GitHub set status to `closed`|
|-|-|

In changeset d76a49ad530ca1269d9d62eedcc9d787ad38aede
```
#!CommitTicketReference repository="" revision="d76a49ad530ca1269d9d62eedcc9d787ad38aede"
Merge pull request #439 from twisted/8702-rodrigc-imp

Author: rodrigc
Reviewer: alex
Fixes: #8702

Remove use of deprecated "import imp" in runtime.py
```

**Comment by twisted-trac** [comment]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)<a name="note_3"></a>|@hawkowl commented|
|-|-|

[mass edit] Removing review from closed tickets.
